index.js

```diff
@@ -45,7 +45,10 @@ module.exports = function lint(text, opts = {}) {
       }
     })
     .join('');
-  return check(js);
+  const checkOptions = {
+    allowAwaitOutsideFunction: opts.await || false,
```

Comment:
Can just cast to boolean here.
```suggestion
    allowAwaitOutsideFunction: !!opts.await,
```